src/main/java/bio/terra/cloudfiletodatastore/GsonConverter.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.cloudfiletodatastore;
+
+import bio.terra.cloudfunctions.common.GsonWrapper;
+import com.google.gson.reflect.TypeToken;
+import java.util.List;
+
+public class GsonConverter {
```

Comment:
These methods can be refactored into `GsonWrapper` instead of having a separate `GsonConverter` class.